components/checkbox/overview.md

```diff
@@ -31,23 +33,42 @@ The <a href="https://www.telerik.com/blazor-ui/checkbox" target="_blank">Blazor
 }
 ````
 
-## Features
+## Indeterminate State
+
+In addition to checked and unchecked basic states, the Blazor CheckBox has a third state - Indeterminate. [Read more about the Blazor Checkbox third state]({%slug checkbox-indeterminate-state%}).
+
+## Appearance
+
+The Checkbox component provides size and border settings to control its appearance. [Read more about the Blazor Checkbox appearance settings]({%slug checkbox-appearance%}).
+
+## Events
 
-The CheckBox provides the following features:
+The Blazor Checkbox fires value change, focus and state change events that you can handle and further customize its behavior. [Read more about the Blazor Checkbox events]({%slug checkbox-events%}).
 
-* `Class` - the CSS class that will be rendered on the main wrapping element of the CheckBox.
-* `Enabled` - whether the component is enabled.
-* `Id` - renders as the `id` attribute on the `<input />` element, so you can attach a `<label for="">` to it.
-* `TabIndex` - the `tabindex` attribute rendered on the CheckBox.
-* `Value` and `bind-Value`- mapped to the `Checked` property of the normal HTML checkbox
-  * The `Value` and `bind-Value` accept `bool` and `bool?` types
-* `Indeterminate` and `bind-Indeterminate` - see the [Indeterminate state]({%slug checkbox-indeterminate-state%}) article for more information and examples
-* Events - see the [CheckBox events]({%slug checkbox-events%}) article for more information and examples.
-* Validation - see the [Input Validation]({%slug common-features/input-validation%}) article for more details.
+## Checkbox Parameters
 
-## Examples
+The Blazor CheckBox provides various parameters that allow you to configure the component:
 
->caption Example that showcases the "I agree to the terms and conditions" basic scenario
+<style>
+    article style + table {
+        table-layout: auto;
+        word-break: normal;
+    }
+</style>
+| Parameter | Type and Default Value | Description |
```

Comment:
* Sort parameters alphabetically.
* Provide information where the Class is rendered;
* Remove "Default Value" from header cell;
* Explain briefly what the Indeterminate parameter does. Then link.